test/cicd-pipeline-stack.test.ts

```diff
@@ -0,0 +1,282 @@
+import * as cdk from "aws-cdk-lib";
+import { Match, Template } from "aws-cdk-lib/assertions";
+import { ShellStep } from "aws-cdk-lib/pipelines";
+import {
+  CICDPipelineStack,
+  getCodeartifactPublishAction,
+} from "../src";
+
+test("Basic CICDPipeline", () => {
```

Comment:
My ESLint is very unhappy about the use of double quotes for string here :) To be honest I have no strong opinions on double quotes vs single quotes, but we should be consistent to whichever we pick.